examples/tests/sdk_tests/std/map.test.w

```diff
@@ -71,26 +71,26 @@ test "size()" {
 let greeting = { "hello" => "there!" };
 assert(greeting.get("hello") == "there!");
 
-let general: str? = greeting.get("grievous");
+let general: str? = greeting.tryGet("grievous");
```

Comment:
Where are the `get` tests?

Reply:
There we already a bunch of existing ones in: examples/tests/valid/container_types.test.w

Reply:
The tests under `valid/` are not "official" tests. We should aim to have coverage for all of our SDK APIs under the `sdk_tests/` tree.

Reply:
Ah okay. I guess I was still under the impression everything in our `valid` was "official" but yea this file already has happy path tests for `.get` I added some unhappy path tests in the unofficial files Ill just move those